### PR TITLE
misc(logger): add time/timeEnd methods

### PR DESCRIPTION
**Pull request by patrickhulce**
In order to land #3745, we need logger to be published with the new functionality. This is just the `lighthouse-logger` portion of #3745 with a version bump to `1.1.0`

### CODE REVIEW

File: lighthouse-logger/index.js [hunk 1]
```diff
@@ -6,6 +6,8 @@
 'use strict';
 
 const debug = require('debug');
+const marky = require('marky');
```
Comment (wardpeet) [inline lighthouse-logger/index.js]: 👌

File: lighthouse-logger/index.js [hunk 2]
```diff
@@ -104,6 +106,16 @@ class Log {
...
   }
 
+  static time({msg, id, args=[]}, level='log') {
+    marky.mark(id);
+    Log[level]('status', msg, ...args);
+  }
+
+  static timeEnd({msg, id, args=[]}, level='verbose') {
+    Log[level]('statusEnd', msg, ...args);
+    return marky.stop(id);
```
Comment (brendankenny) [inline lighthouse-logger/index.js]: is this going to be used? Should update `lighthouse-logger/index.d.ts` or drop the `return` here?

File: lighthouse-logger/index.js [hunk 3]
```diff
@@ -207,5 +219,10 @@ class Log {
 }
 
 Log.events = new Emitter();
+Log.getEntries = _ => {
```
Comment (brendankenny) [inline lighthouse-logger/index.js]: I realize this (appears) to be going for the same `getEntries()` name as the `Performance.getEntries()`, but it's definitely confusing in the context of a logger (where you might expect it to return all log entries).

WDYT about `Log.getTimeEntries()`?

Comment (wardpeet) [inline lighthouse-logger/index.js]: maybe make this a getter instead of a function. No big deal though

Comment (patrickhulce) [reply lighthouse-logger/index.js]: I had the same feeling as well, will do 👍

File: lighthouse-logger/index.js (hunk 2 above)
Comment (patrickhulce) [reply lighthouse-logger/index.js]: done

File: lighthouse-logger/index.js (hunk 3 above)
Comment (patrickhulce) [reply lighthouse-logger/index.js]: settled on takeTimeEntries to mimic the semantics and name of PerfObserver 👍

**Pull request merged**